casepropods/family_connect_subscription/plugin.py

```diff
@@ -61,8 +61,9 @@ def read_data(self, params):
                 subscription_data['rows'].append({
                     "name": "Message Set", "value": message_set["short_name"]})
                 if subscription['active'] and message_set['next_set']:
-                    actions.append(self.get_messageset_action(
-                        message_set, subscription['id']))
+                    subscription_data['actions'].append(
+                        self.get_messageset_action(
+                            message_set, subscription['id']))
```

Comment:
I'm wondering what will happen here if this is the last messageset, and `next_set` is `None`?